proto/testing/ssz_static_test.go

```diff
@@ -1,152 +0,0 @@
-package testing
```

Comment:
Why delete this? We still want to make sure our spec objects SSZ correctly. Restore this file and the minimal and mainnet tests.